### PR TITLE
Semaphore CI

### PR DESCRIPTION
**Pull request by ignatenkobrain**
cc @markoa

### CODE REVIEW

File: ci/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,49 @@
+FROM fedora
+MAINTAINER Igor Gnatenko <i.gnatenko.brain@gmail.com>
+
+WORKDIR /opt/rpm
+COPY . .
+
+RUN echo -e "deltarpm=0\ninstall_weak_deps=0\ntsflags=nodocs" >> /etc/dnf/dnf.conf
+RUN dnf -y update
+RUN dnf -y install \
```
Comment (cgwalters) [inline ci/Dockerfile]: In some [PAPR](https://github.com/projectatomic/papr) projects like rpm-ostree we just pull the builddeps from the distro, I think this is a lot saner than repeating them here again.  See [this example](https://github.com/projectatomic/rpm-ostree/blob/f25444554dac95c5ff469f64192bad17a2916c3a/ci/build.sh#L20).

Comment (cgwalters) [inline ci/Dockerfile]: Which calls into this: https://github.com/projectatomic/rpm-ostree/blob/f25444554dac95c5ff469f64192bad17a2916c3a/ci/libbuild.sh#L18

(Which is a good example of a minor annoyance from the yum → dnf rename)

Comment (ignatenkobrain) [inline ci/Dockerfile]: well, yes and no. 

* If RPM starts/stops requiring something -- it is only way how to properly deal with that (especially with RPM release cadence)
* We don't really need all deps for all tests here

**Comment by markoa** [comment]
LGTM :shipit:

**Comment by ignatenkobrain** [comment]
One thing I'm wondering about, why I don't see job triggered by this pull-request ;)

**Pull request merged**